agent/06_agent_create_cluster.sh

```diff
@@ -163,31 +169,35 @@ function mce_complete_deployment() {
   mce_apply_postinstallation_manifests ${mceManifests}
 }
 
-create_image
-if [[ "${AGENT_DISABLE_AUTOMATED:-}" == "true" ]]; then
-  disable_automated_installation
-fi
+if [[ "${BOOT_MODE}" == "PXE" ]]; then
+  create_pxe_files
+else
+  create_image
+  if [[ "${AGENT_DISABLE_AUTOMATED:-}" == "true" ]]; then
+    disable_automated_installation
+  fi
 
-attach_agent_iso master $NUM_MASTERS
-attach_agent_iso worker $NUM_WORKERS
+  attach_agent_iso master $NUM_MASTERS
+  attach_agent_iso worker $NUM_WORKERS
```

Comment:
Wondering why these additional spaces were added here and below

Reply:
Just because of indentation as the create_image and rest of the instructions are in the else block.